src/main/java/org/jabref/logic/importer/AuthorListParser.java

```diff
@@ -100,6 +100,7 @@ private record SimpleNormalFormResult(String authors, boolean andOthersPresent)
 
     private static SimpleNormalFormResult getSimpleNormalForm(String listOfNames) {
         listOfNames = listOfNames.replace(" -", "-").trim();
+        listOfNames = listOfNames.replaceAll("\\s{2,}", " and ");
```

Comment:
If there is a a single space AND multiple spaces, replace the multiple spaces by " and ".

Meaning: Cover this replacement by an `if`. -- Your test case also should go through